Documentation/config/commitgraph.adoc

```diff
@@ -8,6 +8,14 @@ commitGraph.maxNewFilters::
 	Specifies the default value for the `--max-new-filters` option of `git
 	commit-graph write` (c.f., linkgit:git-commit-graph[1]).
 
+commitGraph.changedPaths::
+	If true, then `git commit-graph write` will compute and write
+	changed-path Bloom filters by default, equivalent to passing
+	`--changed-paths`. If false or unset, existing Bloom filters will persist,
+	while new ones will only be written when explicitly requested via
+	`--changed-paths`. Command-line option `--changed-paths` always takes
```

Comment:
I would replace `--changed-paths` with `--[no-]changed-paths` to signal that either mode will override the config option.